frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java

```diff
@@ -185,18 +185,24 @@ private static String addNameNodeBootstrapCommands(int instanceIndex) {
         if (instanceIndex == 0) {
             return String.format("echo 'Formating filesystem';" +
                     "./%s/bin/hdfs namenode -format -force && " +
+                    "echo 'ZKFC format';" +
+                    "./%s/bin/hdfs zkfc -formatZK -force && " +
                     "echo 'Initialize Shared Edits' && " +
                     "./%s/bin/hdfs namenode -initializeSharedEdits -force && " +
+                    "echo 'Starting ZKFC process';" +
+                    "./%s/bin/hdfs zkfc & " +
```

Comment:
Note that both ZKFC and NameNode processes need to be running in parallel, hence sending the `zkfc` process to the background.